public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -69,7 +72,7 @@ export async function fetchTenantNameList(http: HttpStart): Promise<string[]> {
 
 export function transformTenantData(
   rawTenantData: DataObject<Tenant>,
-  isPrivateEnabled: boolean
+  isPrivateEnabled?: boolean
```

Comment:
Please remove this argument if no longer needed.

Reply:
Cool. Thanks.